uaclient/api/api.py

```diff
@@ -35,6 +36,9 @@
 def call_api(
     endpoint_path: str, options: List[str], cfg: UAConfig
 ) -> APIResponse:
+    # setup null handler for all API endpoints
+    logging.getLogger("uaclient").addHandler(logging.NullHandler())
```

Comment:
We want this to take affect when another project imports something from `uaclient.api`, so I think we want this at the module level in `uaclient/api/__init__.py`